### PR TITLE
Fix for issue #73

### PR DESCRIPTION
**Pull request by alexellis**
## Description

Fixes issue found in #73 via @bmcstdio 

## How Has This Been Tested?

* Set an invalid value for `faas_gateway_address` such as `gateway.openfaas.svc.cluster.local:9090` - at runtime `:8080` will be appended and this will now fail to post but not cause a panic
* Set an invalid value for `faas_gateway_port` and the code will panic on start-up and log out the error

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
Feel free to test out `openfaas/queue-worker:0.8.4-rc9-amd64`

**Comment by alexellis** [comment]
cc @acornies @jonatasbaldin please take a look at the changes mostly from @bmcstdio with some additional missing error handling (which may be historical).

I'm curious why we didn't run into this earlier, perhaps the image wasn't updated since we accepted the changes?

File: main.go [hunk 1]
```diff
@@ -157,51 +140,50 @@ func main() {
...
 		}
 
 	}
 
-	natsURL := fmt.Sprintf("nats://%s:%d", config.NatsAddress, config.NatsPort)
+	natsURL := "nats://" + config.NatsAddress + ":4222"
 
 	natsQueue := NATSQueue{
-		clusterID: config.NatsClusterName,
+		clusterID: "faas-cluster",
```
Comment (bmcstdio) [inline main.go]: Regression?

**Comment by alexellis** [comment]
Squashed / rebased. Latest image is: `openfaas/queue-worker:0.8.4-rc9-amd64`

Anyone affected by this can revert to an earlier version of the queue-worker. Before https://github.com/openfaas/nats-queue-worker/pull/67 was merged or try the pre-release image from above.

**Comment by bmcstdio** [comment]
@alexellis `openfaas/queue-worker:0.8.4-rc9-amd64` seems to be working fine regarding this issue.

**Comment by alexellis** [comment]
Thanks Bruno. I'll kick off a release, can you update helm please?

**Comment by alexellis** [comment]
https://github.com/openfaas/nats-queue-worker/releases/tag/0.8.4

**Comment by bmcstdio** [comment]
https://github.com/openfaas/faas-netes/pull/556

**Pull request merged**